### PR TITLE
Change text in Nick.text

### PR DESCRIPTION
**Pull request by kraftmatic**
Open in text editor and check for misspellings.

### CODE REVIEW

File: Nick.txt [hunk 1]
```diff
@@ -1 +1 @@
-My name is Nick and I work at EHI Holdings, Inc.
+More work. My name is Nick and I work at EHI Holdings, Inc.
```
Comment (kraftmatic) [inline Nick.txt]: You used two spaces after a period.  WTF?

**Comment by kraftmatic** [comment]
@bcalisch Go ahead and rereview.

**Comment by bcalisch** [comment]
Looks good.

**Comment by kraftmatic** [comment]
:+1:

**Pull request merged**